### ISSUE TITLE
Linear activation function interaction with ROC_AUC metric

### ISSUE THREAD

**Issue by Aidan-Kehoe** [rusty1s/himp-gnn#9]
I see in the tox21 example , the evaluation metric is the roc_auc_score on the multilabel binary classification task. However, the final activation function in the model is linear. Does this output interact well with the roc_auc_score? I believe the sklearn implementation of the metric expects an array of shape (n_samples, n_classes) where the values in the array correspond to probability estimates for the labels in each class. As I understand, the current output of the model does not correspond to a probability estimate (linear activation), so does this pose an issue for getting the correct roc_auc value? Would a better implementation be with a sigmoid activation function at the output layer?
Thanks

**Comment by rusty1s** [comment]
As far as I see, this is not a problem since `roc_auc_score` works for both probability estimates and non-thresholded decision values, see [here](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.roc_auc_score.html).
